services/minespace-web/common/components/tailings/AssociatedDams.tsx

```diff
@@ -46,6 +49,27 @@ const AssociatedDams: FC<AssociatedDamsProps> = (props) => {
     history.push(url);
   };
 
+  const editViewIcon = canEditTSF ? (
+    <img src={EDIT_OUTLINE} className="icon-sm padding-sm--right primary-colour" />
```

Comment:
Do people who have edit access not have the ability to just view the record?

Reply:
This has been adjusted now where people with edit access can choose to view or edit.